app/bingey-ui/src/components/WatchlistsControler/WatchlistPicker/WatchlistPicker.jsx

```diff
@@ -43,22 +31,33 @@ export const WatchlistPicker = withRouter(
           {open ? <ExpandLess /> : <ExpandMore />}
         </ListItem>
         <Collapse in={open} timeout='auto' unmountOnExit>
-          {watchlists.map((watchlist) => (
-            <ListItem
-              button
-              key={watchlist._id}
-              className={
-                history.location.pathname.includes(watchlist._id)
-                  ? 'item-text selected'
-                  : 'item-text'
-              }
-              onClick={() => {
-                history.push(`/watchlists/${watchlist._id}`);
-              }}
+          {watchlists.length === 0 ? (
+            <Typography
+              className='center'
+              display='block'
+              variant='caption'
+              gutterBottom
             >
-              {watchlist.name}
-            </ListItem>
-          ))}
+              You currently have no watchlists
+            </Typography>
+          ) : (
+            watchlists.map((watchlist) => (
+              <ListItem
+                button
+                key={watchlist._id}
+                className={
+                  history.location.pathname.includes(watchlist._id)
+                    ? 'item-text selected'
+                    : 'item-text'
+                }
+                onClick={() => {
+                  history.push(`/watchlists/${watchlist._id}`);
+                }}
+              >
+                {watchlist.name}
```

Comment:
I think using `ListItemText` would look nicer, the `inset` prop gives a quick and easy way to align them for example
```suggestion
                <ListItemText
                    primary={watchlist.name}
                    disableTypography
                    inset
                    className='item-text'
                  />
```

the `disableTypography` is needed here to achieve the overflow-ellipses and selected highlight

Reply:
I prefer the other way, there are many places where we use Typography and this barely saves any code

Reply:
As it is now, the watchlists are hierarchically on the same indentation as the drawer's _'Watchlists' folder_ item which makes them seem out of place. The `inset` prop on `ListItemText` solves this easily